policies/optimized-images.md

```diff
@@ -14,19 +14,42 @@ Optimized image policies are aiming to solve problems with sites shipping images
 
 ## What are "Optimized Image Policies"?
 
-Optimized image policies introduce a set of restrictions (policies) on images that can be applied with dev-time enforcement. An image will be rendered as a **placeholder image** when violating a policy, making it easy for web developers to identify and fix the error.
+Optimized image policies introduce a set of restrictions (policies) on images that can be applied with dev-time enforcement. An image will be replaced with a **placeholder image** when violating a policy, making it easy for web developers to identify and fix the error. In addition, violations can be observed via [Feature Policy Reporting](https://github.com/w3c/webappsec-feature-policy/blob/master/reporting.md). With HTTP header `Feature-Policy-Report-Only` (enabled via Feature Policy Reporting origin trial), image policies can be specified in "report-only" mode, which means images will render normally without enforcement, but policy violations will be reported, like [Content Security Policy](https://w3c.github.io/webappsec-csp/#cspro-header).
 
 
 ### Optimized image policies
 
 *   **["oversized-images" policy](#oversized-images)**
-    *   Images must not be bigger than its container size by more than _***X times***_ .
-*   **["unoptimized-images" policy](#unoptimized-images)**
-    *   Images used in rendering must not include too much metadata.
-    *   Images must not be more than _***X bits***_ per compressed pixel.
-    *   Images should be in one of the modern image formats that yield large byte savings and performance improvement.
+    *   The intrinsic dimensions of `<img>` elements must not be larger than the container size by more than _***X times***_.
+*   **["unoptimized-lossy-images" policy](#unoptimized-{lossy,lossless}-images)**
+    *   Lossy images used in rendering must not include too much metadata.
+    *   Lossy images should be in one of the modern image formats that yield large byte savings and performance improvement.
+    *   A lossy `<img>` element should not exceed a byte-per-pixel ratio of _***X***_, with a fixed _**1KB**_ overhead allowance. For a W x H image, the file size threshold is calculated as `W x H x X + 1024` (where X is specified in the policy). Any image whose file size exeeds the limit will be blocked.
+*   **["unoptimized-lossless-images" policy](#unoptimized-{lossy,lossless}-images)**
+    *   Lossless images used in rendering must not include too much metadata.
+    *   Lossless images should be in one of the modern image formats that yield large byte savings and performance improvement.
+    *   A lossless `<img>` element should not exceed a byte-per-pixel ratio of _***X***_, with a fixed _**1KB**_ overhead allowance. For a W x H image, the file size threshold is calculated as `W x H x X + 1024` (where X is specified in the policy). Any image whose file size exeeds the limit will be blocked.    
```

Comment:
Maybe make a note here, or in the origin trial section, that we are experimenting with an alternate number of bytes for overhead?

Also, I think that the overhead allowed for `unoptimized-lossless-images` is 10k currently, and 1k is for `unoptimized-lossless-images-strict`, but check with @paulmeyer90, or take a look in the code.